manifest.json

```diff
@@ -14,14 +14,19 @@
         "type": "number"
       },
       "evmPrivateEncrypted": {
-        "description": "The encrypted key to use for permit generation",
+        "description": "The encrypted key to use for permit generation and auto transfers",
         "examples": ["0x000..."],
         "type": "string"
       },
       "erc20RewardToken": {
         "default": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
         "type": "string"
       },
+      "automaticTransferMode": {
+        "default": true,
+        "description": "If set to false or if there are insufficient funds to settle the payment, permits will be generated instead of processing direct payouts.",
```

Comment:
```suggestion
        "description": "If set to false, or if there are insufficient funds to settle the payment, permits will be generated instead of transferring rewards to the benificiaries.",
```